packages/core/src/selectors/prompts.ts

```diff
@@ -0,0 +1,28 @@
+export const getDefaultSingleSelectPrompt = (
```

Comment:
Prompts in LITS are functions, but we don't need the get prefix for them. I would also export a type to make it easier for the user to replace the prompt with something else. See the main prompts.ts file for examples.